Source/WebKit/UIProcess/API/glib/WebKitSettings.h.in

```diff
@@ -540,6 +540,21 @@ webkit_settings_get_disable_web_security                       (WebKitSettings *
 WEBKIT_API void
 webkit_settings_set_disable_web_security                       (WebKitSettings *settings,
                                                                 gboolean        disabled);
+
+typedef struct _WebKitFeature WebKitFeature;
+
+WEBKIT_API void
+webkit_settings_set_feature_enabled                            (WebKitSettings *settings,
+                                                                WebKitFeature  *feature,
+                                                                gboolean        enabled);
+
+WEBKIT_API gboolean
+webkit_settings_get_feature_enabled                            (WebKitSettings *settings,
+                                                                WebKitFeature  *feature);
+
+WEBKIT_API GPtrArray *
```

Comment:
[This one needs a rethink](https://www.bassi.io/articles/2023/02/20/bindable-api-2023/)

"Whatever you do, however low you feel on the day, whatever particular tragedy befell your family at some point, please: never use GLib array types in your API. Nothing good will ever come of it, and you’ll just spend your days regretting this choice."

Better options:

 * Return a WebKitFeature** and warn that each element needs to be deleted with webkit_feature_unref() then the container deleted with g_free()
 * Create a WebKitFeatureList type

Reply:
Ahhh, that's why I was getting the back of my head a bit itchy while writing `GPtrArray`... I think we can have a new `WebKitFeatureList` the same way we have `WebKitBackForwardList` and GLib has `GUnixFDList`.

(While on topic, it looks like we have a few functions return `GList`, like `webkit_context_menu_get_items`, `webkit_cookie_manager_get_cookies_finish`, etc. and maybe it was an oversight to let them be while working on the 2022 API 😨)